### DISCUSSION TITLE
https://angular.love/all-you-need-to-know-to-jumpstart-with-ngxs

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#355] [General]
# https://angular.love/all-you-need-to-know-to-jumpstart-with-ngxs

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/all-you-need-to-know-to-jumpstart-with-ngxs

**Comment by RichardBoyder** [comment]
lovely Tutorial. Could it if possible be updated with latest angualr and latest ngxs? would be great having a refreshed up to date Tutorial like the above?
